### PR TITLE
[16.0] [FIX] web_chatter_position: fixes JS assuming "o_form_sheet_bg" is always found

### PR DESCRIPTION
**Pull request by BT-jmontero**
Fixes the issue reported here: https://github.com/OCA/web/issues/2464

### CODE REVIEW

File: web_chatter_position/static/src/js/web_chatter_position.esm.js [hunk 1]
```diff
@@ -126,15 +126,19 @@ patch(FormCompiler.prototype, "web_chatter_position", {
...
+                    sheetBgChatterContainerHookXml.classList.add("o-isInFormSheetBg");
+                    sheetBgChatterContainerHookXml.setAttribute("t-if", true);
+                    append(formSheetBgXml, sheetBgChatterContainerHookXml);
+                    const sheetBgChatterContainerXml =
+                        sheetBgChatterContainerHookXml.querySelector(
+                            "ChatterContainer"
+                        );
+                    sheetBgChatterContainerXml.setAttribute("isInFormSheetBg", "true");
+                    chatterContainerHookXml.setAttribute("t-if", false);
+                }
```
Comment (BT-anieto) [inline web_chatter_position/static/src/js/web_chatter_position.esm.js]:
```suggestion
                if (!formSheetBgXml) {
                    return res;
                }
                const sheetBgChatterContainerHookXml =
                    chatterContainerHookXml.cloneNode(true);
                sheetBgChatterContainerHookXml.classList.add("o-isInFormSheetBg");
                sheetBgChatterContainerHookXml.setAttribute("t-if", true);
                append(formSheetBgXml, sheetBgChatterContainerHookXml);
                const sheetBgChatterContainerXml =
                    sheetBgChatterContainerHookXml.querySelector("ChatterContainer");
                sheetBgChatterContainerXml.setAttribute("isInFormSheetBg", "true");
                chatterContainerHookXml.setAttribute("t-if", false);
```

**Review by BT-anieto — COMMENTED** [review]
LGTM, but I added a suggestion in case you want to consider it.

**Review by BT-anieto — APPROVED** [review]

**Comment by BT-jmontero** [comment]
@JuMiSanAr @ivantodorovich As contributors of web_chat_positions, maybe you can review this PR fixing a small issue ;) Thanks in advance for your time!!

File: web_chatter_position/static/src/js/web_chatter_position.esm.js (hunk 1 above)
Comment (JuMiSanAr) [reply web_chatter_position/static/src/js/web_chatter_position.esm.js]: I agree with this suggestion :+1:

**Review by JuMiSanAr — APPROVED** [review]

**Review by ivantodorovich — APPROVED** [review]
Agree w/ the suggestion :) pre-approving

**Comment by BT-jmontero** [comment]
Hola @yajo, this module has no maintainer. Could please merge this PR? Thanks in advance for your support!!

**Comment by yajo** [comment]
You can volunteer to become a maintainer if you want 😉 

/ocabot merge patch

**Pull request merged**